.github/workflows/pinot_tests.yml

```diff
@@ -58,6 +58,7 @@ jobs:
           cache: 'maven'
       - name: Linter Test
         env:
+          GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GE_ACCESS_TOKEN }}
```

Comment:
Is this `GE_ACCESS_TOKEN` an apache org secret?

Reply:
Correct, it is an apache org secret managed by the infra team